docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Why do we need a .bat file? I never intend to generate this on Windows :)

Reply:
This is added by sphinx-quickstart :) It generates a two Makefiles and a conf.py. Let me get rid of this.